### ISSUE TITLE
Error: net::ERR_NETWORK_CHANGED

### ISSUE THREAD

**Issue by andregce** [wulkano/Kap#1080]
**macOS version:**    Big Sur (11.6.1)
**Kap version:**      3.4.2

Unhandled Promise Rejection
Error: net::ERR_NETWORK_CHANGED
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:101:6400)

#### Steps to reproduce
I woke up my computer and connected to a VPN. My guess is that Kap was trying to do something and got caught by the network change.

#### Current behavior
Kap throws an error when switching networks (sometimes). It does not crash, just throws the error.

#### Expected behavior
No error pops up while connecting to a VPN.

**Comment by gillisig** [comment]
Downloads are broken as well. 500 Error.

**Comment by gillisig** [comment]
@timothyis yeah sorry that was a bit rushed from me. But it was on the website, getkao.co but now it seems to be working.
